usagov/vote-gov-drupal
# VOTE-914 Enable and configure TMGMT module for translation management

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-914

## Description (optional)
Install and configure the basic setting to manage translations using the TMGMT module

## Deployment and testing (required, if applicable)
### Post-deploy steps
1. run `lando retune`

### Testing steps
1. Go to the content listing page `/admin/content`
2. Translate an existing content item
3. Select the translation using the checkbox to the name and press the button labeled "Request Translation"
4. Set export to XLIFF and click the button labeled "Submit to Provider"
5. Go to `/admin/tmgmt/jobs` to see a list of open jobs
6. Click on button labeled "manage" in the row of your job
7. Scroll to bottom of page and click on the link labeled "here" in the message "Exported file can be downloaded here."
8. Confirm that the XML in the file represented the content of the node
